docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -414,6 +414,28 @@ All named curves are instances of :class:`EllipticCurve`.
     SECG curve ``secp256k1``.
 
 
+.. class:: BrainpoolP256R1
+
+    .. versionadded:: 2.2
+
+    Brainpool curve specified in :rfc:`5639`. These curves are not commonly
+    used but are available for interoperating with existing systems.
```

Comment:
Let's go stronger: These curves are discouraged for new systems, in favor of X25519, but...